drivers/hmis/app/graphql/types/hmis_schema/ce_referral_source_household_member.rb

```diff
@@ -21,6 +22,10 @@ class HmisSchema::CeReferralSourceHouseholdMember < Types::BaseObject
       field :can_view_clients, Boolean, null: false
     end
 
+    def client_id
+      client&.id
```

Comment:
safe navigation operator needed because client can be nil? The graphql field is non-null